src/apps/cli/commands/validate.ts

```diff
@@ -16,7 +16,7 @@ import {
 
 export default class Validate extends Command {
   static description = 'validate asyncapi file';
-  private validationService = new ValidationService();
+  private validationService = new ValidationService({}, true);
```

Comment:
This looks really confusing. What's that empty object and that `true` value? What do they mean?

My suggestion is that you merge the two. If you still want to keep parserOptions separated, in the ValidationService, you can do something like:

```js
{
  parserOptions: ...,
  customAuth: true
}
```

This way it'll be much clearer what these things mean.

Now that said, is there any case we should not be using the customAuth in the CLI? Shouldn't it be always enabled? Also, the parserOptions object already have a way to provide the custom resolver, why do we need another parameter? 🤔